Library/Homebrew/cmd/tap.rb

```diff
@@ -1,17 +1,15 @@
 module Homebrew
   def tap
     if ARGV.empty?
-      each_tap do |user, repo|
-        puts "#{user.basename}/#{repo.basename.sub("homebrew-", "")}" if (repo/".git").directory?
-      end
+      show_taps
     elsif ARGV.first == "--repair"
       migrate_taps :force => true
     else
-      opoo "Already tapped!" unless install_tap(*tap_args)
+      opoo "Already tapped!" unless install_tap(*tap_args, ARGV.named[1])
     end
   end
 
-  def install_tap user, repo
+  def install_tap user, repo, clone_target
```

Comment:
`def install_tap user, repo, clone_target=nil`

So it can accept two arguments as this method is also used elsewhere.